clicommand/pipeline_upload.go

```diff
@@ -131,14 +134,14 @@ var PipelineUploadCommand = cli.Command{
 			Usage:  "When true, fail the pipeline upload early if the pipeline contains secrets",
 			EnvVar: "BUILDKITE_AGENT_PIPELINE_UPLOAD_REJECT_SECRETS",
 		},
-		cli.BoolFlag{
+		cli.BoolTFlag{
 			Name:   "apply-if-changed",
 			Usage:  "When enabled, steps containing an ′if_changed′ key are evaluated against the git diff. If the ′if_changed′ glob pattern match no files changed in the build, the step is skipped.",
 			EnvVar: "BUILDKITE_AGENT_APPLY_SKIP_IF_UNCHANGED",
 		},
 		cli.StringFlag{
 			Name:   "git-diff-base",
-			Usage:  "Provides the base from which to find the git diff when processing ′if_changed′",
+			Usage:  "Provides the base from which to find the git diff when processing ′if_changed′, e.g. origin/main",
 			Value:  "origin/main",
```

Comment:
Should we default to:

https://buildkite.com/docs/pipelines/configure/environment-variables#BUILDKITE_PIPELINE_DEFAULT_BRANCH

Reply:
That's probably a more sensible choice, yeah.